src/styles/index.css

```diff
@@ -24,8 +24,28 @@ h1 {
   }
 }
 
-@screen xl {
-  h1 {
-    @apply text-3xl;
-  }
+/* Code block styles */
+.prose pre {
+  @apply mx-2 px-8 py-6 bg-gray-800 text-gray-200 rounded-md;
+}
+
+pre {
+  @apply mx-2 px-8 py-6 bg-gray-800 text-gray-200 rounded-md;
+}
+
+/* Contain typography to constrained width */
+.prose h1, h2, h3, h4, h5, p, ul, ol, li, span {
+  @apply max-w-2xl mx-auto;
 }
+
+h1, h2, h3, h4, h5, p, ul, ol, li, span {
+  @apply max-w-2xl mx-auto;
+}
```

Comment:
**suggestion:** I would prefer setting max width and margin on a container wrapping these general elements (usually layout or template component) rather than global level.

Reply:
Yeah, we can do it at the template/component level - just removed the `.prose` size constraints. Should I nix the non-prose ones too?

Reply:
Nixed: f409cdb